templates/vue/src/components/Lightbox/media/MultiContentMedia/PageNavigationBar/index.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+  <div ref="wrapper" class="page-nav-wrapper">
+    <aside
+      ref="container"
+      data-qa="page-nav-container"
+      :class="[
+        'page-nav',
+        {
+          lightbox: !node.fullscreen,
+          closed: burgerView && !opened,
+        },
+      ]"
+      :style="pageNavStyle"
+    >
+      <button
+        v-if="burgerView"
+        class="page-nav-toggle"
+        data-qa="page-nav-toggle"
+        @click="opened = !opened"
+      >
+        <i v-if="!opened" class="fas fa-bars fa-lg"></i>
+        <i v-else class="fas fa-times fa-lg"></i>
+      </button>
+      <div v-if="opened || !burgerView" class="page-nav-content">
+        <page-menu
+          v-for="row in rows"
+          :key="row.node.id"
+          :node="row.node"
+          :active="Number(active)"
+          :lockRows="lockRows"
+          :shouldDisable="disableRow(row.node.id)"
+          :isBase="true"
+        />
+      </div>
+    </aside>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from "vuex"
+import PageMenu from "./PageMenu"
+import Helpers from "@/utils/Helpers"
+
+export default {
+  name: "page-navigation-bar",
+  components: {
+    PageMenu,
+  },
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+    parentRefs: {
+      type: Object,
+      required: true,
+    },
+    dimensions: {
+      type: Object,
+      required: false,
+      default: null,
+    },
+  },
+  data() {
+    return {
+      opened: false,
+      browserWidth: Helpers.getBrowserWidth(),
+    }
+  },
+  created() {
+    window.addEventListener("resize", Helpers.debounce(this.setBrowserWidth, 300))
+  },

```

Comment:
Correct me if I'm wrong @shirleywbi, but from skimming the code it looks like the resize event is being used to size the navbar and the node contents? If so, this might be what's causing the full screen bug (because when you go full screen, the resize event doesn't trigger) and the weird resizing behaviour.

I think you could solve this UI requirement using CSS alone, which would make the code a lot simpler and offload the layout work to the browser. Here's a small example I made:

https://codesandbox.io/s/async-currying-fx4nk?file=/index.html

I might be missing something here, so I do apologize if you've already tried approach!

Reply:
Good call. I was trying to do something similar but was having a lot of issues with `position: absolute`. There was some alignment in the code elsewhere that was making it behave differently. Should be better now. Thanks.